plugins/scanners/scancode/src/test/assets/scancode-output-format-2.0.0_mime-types-2.1.18.json

```diff
@@ -13,25 +13,27 @@
         "--license": true,
         "--processes": "3",
         "--strip-root": true,
-        "--timeout": "300.0"
+        "--timeout": "0.09"
```

Comment:
Any reason why you using 0.09 here but 0.07 above? Similar for the next file.

Reply:
I played with the values until I got exactly one issue.